frontend/app/views/spree/home/_carousel.html.erb

```diff
@@ -6,92 +6,103 @@
           <div class="row">
             <% sliced_items.each do |item| %>
               <div class="col-6">
+                <% if item == sliced_items.first %>
```

Comment:
It'd be cleaner to use `each_with_index` and compare indices instead of using object equality.